docs/integrations/emr/emr_configuration.md

```diff
@@ -193,7 +184,18 @@ curl -H "Authorization: ApiKey ${API_KEY}" https://$HOST/hopsworks-api/api/proje
 echo -n $(curl -H "Authorization: ApiKey ${API_KEY}" https://$HOST/hopsworks-api/api/project/$PROJECT_ID/credentials | jq -r .password) > material_passwd
 
 chmod -R o-rwx /usr/lib/hopsworks
+
+sudo pip3 install --upgrade hsfs~=X.X.0
 ```
+!!! note
+    Don't forget to replace X.X.0 with the major and minor version of your Hopsworks deployment.
+
+<p align="center">
+    <figure>
+        <img src="../../../assets/images/hopsworks-version.png" alt="HSFS version needs to match the major version of Hopsworks">
+        <figcaption>You find the Hopsworks version inside any of your Project's settings tab on Hopsworks</figcaption>
```

Comment:
To find your Hopsworks version, enter any of your projects and go to the settings tab inside your project.

Reply:
Done